app/uk/gov/hmrc/agentsubscription/controllers/UrlBinders.scala

```diff
@@ -14,20 +14,18 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.agentsubscription.model
+package uk.gov.hmrc.agentsubscription.controllers
 
-import org.scalatest.{ShouldMatchers, WordSpec}
+import play.api.mvc.PathBindable
+import uk.gov.hmrc.agentmtdidentifiers.model.Utr
 
-class UtrSpec extends WordSpec with ShouldMatchers {
-
-  "isValid" should {
-    "be true for valid UTRs" in {
-      Utr.isValid("1234567890") shouldBe true
+object UrlBinders {
+  implicit val utrBinder = new PathBindable[Utr] {
```

Comment:
`RegistrationControlerISpec."return 400 when the UTR is invalid"` is failing. One way to fix it is to change this to:
```
implicit val utrBinder = new SimpleObjectBinder[Utr](Utr.apply, _.value)
```